### ISSUE TITLE
Left/Right Indicators for Titles

### ISSUE THREAD

**Issue by JakeWharton** [JakeWharton/ViewPagerIndicator#6]
- `adjacentIndicatorStyle` - enum: 'none', 'fade', 'arrows', 'custom'
- `adjacentIndicatorLeft` - `drawable|reference`, ignored unless 'custom' style
- `adjacentIndicatorRight` - `drawable|reference`, ignored unless 'custom' style

**Comment by franciscojunior** [comment]
Would it be possible to add some fade effect as left/right indicator? 

Just like new android market?

**Comment by JakeWharton** [comment]
Sounds smart. I've updated the description. I'll try to get started on an implementation this week.

**Comment by sibbor** [comment]
Looks like a good project. Was going to write something myself, but you're already made some great progress.
Thank you!

**Comment by virajmody** [comment]
This looks promising - I haven't used it yet. I was on my way to writing something similar when I discovered this. For this particular issue, this writeup might help incase you haven't seen it already: http://www.pushing-pixels.org/2011/08/11/android-tips-and-tricks-swipey-tabs.html

**Comment by JakeWharton** [comment]
@virajmody Check out [the sample APK](https://github.com/JakeWharton/Android-ViewPagerIndicator/Android-ViewPagerIndicator-Sample-1.0.0.apk/qr_code). I've actually already implemented the underline indicator. It doesn't fade the font, but I know how to do that I've just been lacking time to implement it.

**Comment by AfzalivE** [comment]
Also implement that in the tabs when you do Jake.

Also notice how (in Market and Google+ apps) the transitions for the indicator and the page are not the same rate? Would be great if you could do that too!

**Comment by vdesmet93** [comment]
Ok, since this is the issue where I should tell my progess, I'll to post it here.

I made some  improvements to the Left/Right indicators. The default arrow is now drawn on canvas, the color can be changed by both XML and from code. From code, you can use R.color.xxx and Color.BLUE. 

If the developer uses custom mode, they'll have to add their own drawables. The height/width is automatic, but the developer can set a custom width and/or height in both modes. 

There are still some little layout bugs when the user goes to another page. That's what I'm trying to fix in the next few days. If that's fixed I guess it's ready for a pull request.

**Comment by scompt** [comment]
Has there been any progress here regarding faded indicators? It doesn't look like this was ever implemented in @vdesmet93's fork.

**Comment by vdesmet93** [comment]
@scompt That's right, I wasn't able to get it working as it should. If you want, I can push the code so you(or anyone else) can have a look at it?

**Comment by scompt** [comment]
@vdesmet93 Thanks for the offer, but I figured out how to get fading edges working well enough using a commit from #104 and a bit of standard android configuration.
